apps/web/vercel.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://openapi.vercel.sh/vercel.json",
```

Comment:
non-blocking: even with `github.silent: true` in a root file like https://github.com/felicio/status-web/commit/12b34e45fb07e169b4b382bab409dac30803a285 would trigger the bot on unrated PR like https://github.com/felicio/status-web/pull/8. Granted it'd be ignored eventually, but the notifications could get annoying real quick.

Reply:
ditto https://github.com/status-im/status-web/pull/396#discussion_r1192790739 (gh ci/actions)